labtool2.0/src/components/pages/Courses.js

```diff
@@ -17,51 +18,56 @@ export class Courses extends Component {
   render() {
     return (
       <div className="Courses">
-        <Container style={{ overflowX: 'scroll' }}>
+        <Container>
           <Header as="h2">Courses</Header>
           {this.props.loading.loading || !Array.isArray(this.props.courseInstance) ? (
             <Loader active />
           ) : (
-            <Table singleLine color="yellow">
-              <Table.Header>
-                <Table.Row>
-                  <Table.HeaderCell colSpan="1"> </Table.HeaderCell>
-                  <Table.HeaderCell colSpan="1">Course id</Table.HeaderCell>
-                  <Table.HeaderCell colSpan="1">Course name</Table.HeaderCell>
-                  <Table.HeaderCell colSpan="1">Course start date</Table.HeaderCell>
-                  <Table.HeaderCell colSpan="2"> </Table.HeaderCell>
-                </Table.Row>
-              </Table.Header>
+            <HorizontalScrollable>
+              <Table singleLine color="yellow" style={{ overflowX: 'visible' }}>
+                <Table.Header>
+                  <Table.Row>
+                    <Table.HeaderCell colSpan="1"> </Table.HeaderCell>
+                    <Table.HeaderCell colSpan="1">Course id</Table.HeaderCell>
+                    <Table.HeaderCell colSpan="1">Course name</Table.HeaderCell>
+                    <Table.HeaderCell colSpan="1">Course start date</Table.HeaderCell>
+                    <Table.HeaderCell colSpan="2"> </Table.HeaderCell>
+                  </Table.Row>
+                </Table.Header>
 
-              <Table.Body>
-                {this.props.courseInstance.map(instance => (
-                  <Table.Row key={instance.id}>
-                    <Table.Cell>
-                      <div>
-                        {instance.active === true ? (
-                          <Label ribbon style={{ backgroundColor: '#21ba45' }}>
-                            Active registration
-                          </Label>
-                        ) : (
-                          ''
-                        )}
-                      </div>
-                    </Table.Cell>
-                    <Table.Cell>{instance.shorterId} </Table.Cell>
-                    <Table.Cell>
-                      <strong>
-                        <Link to={`/labtool/courses/${instance.ohid}`}>{instance.name}</Link>
-                      </strong>
-                    </Table.Cell>
+                <Table.Body>
```

Comment:
This file seems to have some spacing issues

Reply:
The spacing passes ESLint's prettify as-is

Reply:
Might be a problem with GitHub then